packages/jemalloc/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=http://jemalloc.net/
+TERMUX_PKG_DESCRIPTION="jemalloc is a general purpose malloc(3) implementation that emphasizes
+fragmentation avoidance and scalable concurrency support."
```

Comment:
Short one-line description. No hacks for enabling full multiline description.
Ref: https://github.com/termux/termux-packages/wiki/Creating-new-package#table-of-available-package-control-fields